VMEncryption/main/EncryptionSettingsUtil.py

```diff
@@ -31,7 +33,8 @@ class EncryptionSettingsUtil(object):
 
     def __init__(self, logger):
         self.logger = logger
-        self._CURRENT_DISK_ENCRYPTION_DATA_VERSION = "3.0"
+        self._CURRENT_DISK_ENCRYPTION_DATA_VERSION = "4.0"
```

Comment:
The new section added around 52 in Common.Py references 3.0, not 4.0.  Is that correct?

Reply:
put an example here to clarify which version is it talking about. This one is within the JSON content itself. 

---
In reply to: [273141600](https://github.com/Azure/azure-linux-extensions/pull/780#discussion_r273141600) [](ancestors = 273141600)

Reply:
Yes, these versions are supposed to be different. The one there is for the wire protocol format itself.
This is for the JSON blob's data version